### ISSUE TITLE
Extract text from images embedded in pdfs

### ISSUE THREAD

**Issue by sivi3883** [Azure-Samples/azure-search-openai-demo#399]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ x] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>First of all, thanks for the awesome demo code. Works like a charm in the first attempt. Considering I am new to Azure and CLI commands, the instructions are super clear. 
I am trying to extract texts from the images embedded in PDFs. I believe I could add a skillset for OCR in the search service and rerun the index. 

Looking for your recommendations on how to add it in prepdocs.py. Is there a way to extract the texts from images (as well from the supplied PDFs) through form recognizer? 

### Any log messages given by the failure
>

### Expected/desired behavior
> Use the backend chat application to perform on search text embedded in the images.

### OS and Version?
> Mac OS Monterey

### azd version?
> run `azd version` and copy paste here.

### Versions
>azd version 1.0.2 

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by pamelafox** [comment]
This is still a valid feature request and I have heard it a few times. I have not looked into this myself, however. Leaving it open.

**Comment by egor-yudkin** [comment]
I think that the current DI chunking process already does that. I can see text from embedded images in the index of app we are building. @pamelafox is this somethin you can confirm?

**Comment by pamelafox** [comment]
You're correct, this is already supported in the manual ingestion path: prepdocs uses Azure AI Document Intelligence (prebuilt-layout with high‑resolution OCR) to extract text from images embedded in PDFs. 

To get this behavior, configure AZURE_DOCUMENTINTELLIGENCE_SERVICE in your environment (which is configured by default) and do not set USE_LOCAL_PDF_PARSER (as the local PDF parser does not perform OCR). See docs/data_ingestion.md for details.  If you rely only on integrated vectorization, OCR coverage depends on built-in content extraction; for best results use prepdocs with Document Intelligence.

Closing as addressed—if you still see missing text after enabling DI, please add details and we can reopen.
